bench.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ocaml/opam:debian-ocaml-5.3
+WORKDIR /bench-dir
+RUN sudo apt-get update && \
+    sudo apt-get install -qq -yy --no-install-recommends pkg-config libgmp-dev \
+      wget
+RUN sudo ln -sf /usr/bin/opam-2.1 /usr/bin/opam
```

Comment:
Why do we use an old version of opam ?

Reply:
Good question. I took inspiration from https://github.com/ocaml-multicore/picos/blob/main/bench.Dockerfile which does this. I’ll try without this line.

Reply:
By default opam is even older (2.0). I made it use 2.3 instead.